### DISCUSSION TITLE
Cuy University Semester 4 - Sesi 8

### DISCUSSION THREAD

**Discussion by neet693** [deaafrizal/forum#27] [Troubleshooting Room] [answered]
Permisi suhu - suhu sekalian, di Sesi 8 ini kan belajar tentang AUTH untuk Cuyanimelist sudah berhasil diimplementasikan namun ada bug saat user sudah login dan masuk ke /anime/id ketika mau signout dari halaman tersebeut url btn signout berubah ke /anime/api/auth/signout. Kira kira salah dibagian mana ya? Butuh pencerahan 

Tech stack Next js V.14

**Comment by neet693** [comment, marked as answer]
**Note:**
Sudah tersolved karena salah ketik dan NEXT_URLnya belum di input 🙏

**Reply by deaafrizal** [reply]
GOOD WORK bro!

**Comment by EC-007** [comment]
permisi, disesi 8 pnya gw aneh yah, ketik logout bagian dpn lancar, tpi ketika logout dibagian dashboard "page not found"

**Reply by EC-007** [reply]
![Screenshot (60)](https://github.com/deaaprizal/forum/assets/153619348/820a4987-c8a6-4ee4-ad76-74a81e6010eb)
cari berjam2 hnya karna yg 1 ini

**Comment by Destaanugra1** [comment]
![ai](https://github.com/deaaprizal/forum/assets/152362800/b190a302-6722-473e-8501-15f9c51fc29b)
<div>
      <h1>Paling</h1>
      {anime.data.map(data => {
        return <AnimeList title={data.title} images={data.images.webg.image_url} />
                                                                                                            ^
      })}
    </div>
  )

kenapa error di situ mohon bantuan bg

**Reply by desta-anugra** [reply]
Error di webg.image_url nah error di terminal itu kenapa ya?
